test/adding_tests.md

```diff
@@ -52,12 +52,14 @@ _See [e2e_flags.go](./e2e_flags.go)._
 [When tests are run with `--logverbose`
 option](README.md#output-verbose-logs), debug logs will be emitted to stdout.
 
-We are using [Knative logging library](/pkg/logging) for structured logging, it is built on top of [zap](https://github.com/uber-go/zap).
-Tests should initialize the global logger to use a test specifc context with `test.GetContextLogger`:
+We are using the common [e2e logging library](/pkg/test/logging) that uses the 
```

Comment:
how about "test logging library"

Reply:
This is only for the e2e logging and not all tests. So, I think e2e logging makes more sense here.

Reply:
I was thinking it also covers conformance tests. But it's fine.